### PR TITLE
Support fixtures outside bundles AKA Symfony Flex support

### LINKED ISSUE #296: app/Resources not scanned

### PR DESCRIPTION
**Pull request by dunglas**
Fix #296.

To use it with Flex:

```yml
hautelook_alice:
    fixtures_path: 'fixtures'
```

Then put your fixture files in a `fixtures/` directory at the root of the project.

### CODE REVIEW

File: resources/config/locator.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <service id="hautelook_alice.locator.env_directory"
                  class="Hautelook\AliceBundle\Locator\EnvDirectoryLocator">
             <argument type="string">%hautelook_alice.fixtures_path%</argument>
+            <argument type="string" on-invalid="ignore">%kernel.project_dir%</argument>
```
Comment (theofidry) [inline resources/config/locator.xml]: this is available only since 3.3 right?

Comment (dunglas) [inline resources/config/locator.xml]: Yes but because of the `on-invalid="ignore"`, it will fallback to the previous behavior for older versions.

**Comment by dunglas** [comment]
@theofidry which extra tests do you have in mind?

File: resources/config/locator.xml (hunk 1 above)
Comment (theofidry) [inline resources/config/locator.xml]: If possible it would be nice to support `app/Resources`, so having a `kernel.root_dir` as a fallback would be nice

**Comment by theofidry** [comment]
Mostly testing it under Symfony 3.2 & 3.3 which requires Travis to be updated and the support for `app/Resources`. But maybe we can skip the later

**Comment by mmarchois** [comment]
When this PR will be merged ?

**Comment by theofidry** [comment]
I think running tests against Symfony 3.2 and 3.3 is a requirement for it, so Travis needs to be updated first and then the tests run again for it.

So either when someone will take care of that or when I'll have some time which won't happen before the end of the week/weekend

**Comment by dunglas** [comment]
I may take a look, but do you know why Travis and Scrutinizr don't run at all? Maybe do you need to refresh the access tokens?

**Comment by theofidry** [comment]
I don't think it's the issue, if you check Travis it actually ran yesterday: https://travis-ci.org/hautelook/AliceBundle/pull_requests but for some reason it takes ages and the nothing is returned

**Comment by theofidry** [comment]
Meh, tests seem to be failing on Travis anyway. I'll merge this and fix that later

**Pull request merged**